docs/root/configuration/http/http_filters/bandwidth_limit_filter.rst

```diff
@@ -0,0 +1,100 @@
+.. _config_http_filters_bandwidth_limit:
+
+HTTP Bandwidth limit
+====================
+
+* Bandwidth limiting :ref:`architecture overview <arch_overview_bandwidth_limit>`
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.bandwidth_limit.v3alpha.BandwidthLimit>`
+* This filter should be configured with the name *envoy.filters.http.bandwidth_limit*.
```

Comment:
could you make this an inline literal please - eg ` ``envoy.filters.http.bandwidth_limit`` `